index.html.js

```diff
@@ -8,17 +8,12 @@ const getHTML = (abeUrl, isDev) => {
 </head>
 <body>
 <div id="app"></div>
-<script type="text/javascript">
-    window.abe_url = "${abeUrl}";
-    window.debug = ${isDev.toString()};
-    window.GA_ID = "${process.env.GA_ID}";
-</script>
 <script src="/public/build/bundle.js" type="text/javascript"></script>
 <link rel="stylesheet" href="/node_modules/input-moment/dist/input-moment.css"/>
 <link rel="stylesheet" type="text/css" href="//code.ionicframework.com/ionicons/2.0.1/css/ionicons.min.css">
 <link rel='stylesheet' href='/public/css/app.css'/>
 </body>
-</html>`
-};
+</html>`;
+}
```

Comment:
FYI, the Airbnb style guide suggests semicolons after arrow functions.

Reply:
That wasn't an intentional change. I think my editor removed the first semicolon because I had it set to a different style. Then I turned off format-on-save for this project and manually changed a few places back, but got this one wrong.

Rather than fix it just here, I propose that after the lint fix is in dev, we add the Airbnb preset to the lint config, and then use `eslint --fix` to update this and other areas to the Airbnb style.